### PR TITLE
compute: pick username from template Details

### LINKED ISSUE #96: username from the template details
Feed from the template details to obtain the information regarding the username to use.

### PR DESCRIPTION
**Pull request by greut**
closes #96

### CODE REVIEW

File: exoscale/compute_resource.go [hunk 1]
```diff
@@ -256,6 +262,9 @@ func createCompute(d *schema.ResourceData, meta interface{}) error {
 			continue
 		}
 	}
+	if username == "" {
+		username = getSSHUsername(image)
```
Comment (marcaurele) [inline exoscale/compute_resource.go]: Does this default to `root` ?

Comment (marcaurele) [reply exoscale/compute_resource.go]: You should get rid of `getSSHUsername` and default to `root` in case the username is not in the template details.

Comment (greut) [reply exoscale/compute_resource.go]: I'll keep it for the `readCompute` part of the story though.

**Comment by greut** [comment]
:rocket: thanks for the review!

**Pull request merged**